### DISCUSSION TITLE
I have a problem

### DISCUSSION THREAD

**Discussion by colinleng** [minio/minio#16089] [Q&A] [answered]
Error: Drive /mnt/data/file, path (.minio.sys/buckets/.usage-cache.bin) returned an error (decodeXLHeaders: Unknown xl meta version 2) (*fmt.wrapError)

Error: decodeXLHeaders: Unknown xl meta version 2 (*errors.errorString)

When I started minio, I encountered the above problems，Please tell me how to solve it

**Comment by harshavardhana** [comment, marked as answer]
It looks you first upgraded the server and then downgraded it. This is not allowed
